js/huobi.js

```diff
@@ -2665,40 +2665,41 @@ module.exports = class huobi extends Exchange {
          * @returns {object} an associative dictionary of currencies
          */
         const response = await this.spotPublicGetV2ReferenceCurrencies (params);
```

Comment:
@samgermain precision is a string here, can you fix it?
![image](https://user-images.githubusercontent.com/43336371/185412488-72d13036-07c7-4402-8f96-79bc0f53d138.png)